thorlcr/master/thgraphmanager.cpp

```diff
@@ -1328,7 +1328,7 @@ void thorMain(ILogMsgHandler *logHandler, const char *wuid, const char *graphNam
             else
             {
                 unsigned lingerPeriod = globals->getPropInt("@lingerPeriod", DEFAULT_LINGER_SECS)*1000;
-                bool multiJobLinger = globals->getPropBool("@multiJobLinger");
+                bool multiJobLinger = globals->getPropBool("@multiJobLinger", true);
```

Comment:
What will be the possible effect on mixed version environments?  will it fall-back gracefully?

Reply:
not quite sure what you mean - do you mean if agent was different to thor?

The setting is generated in to the agent settings from the thor: settings.